.github/workflows/report_quiz/report_quiz.sh

```diff
@@ -11,7 +11,7 @@ skipped_quizzes=0
 
 # To escapes some stupid quotes
 escp(){
-    echo -e \"$(printf '%q' "$1")\"
+    echo -e \"$(printf '%q' "$*")\"
```

Comment:
the function escp take just one argument (string). $* take all arguments as one string.
I do this replacement because in the line 62, we cannot use the quote to give a string as a argument